share/completions/fab.fish

```diff
@@ -0,0 +1,75 @@
+# tasks 
+for task in (fab -l)
+    set -l _matched (string match -r "^ +([^ ]*) *([^ ]?.*)" $task)
+
+    if test -n "$_matched"
+        set -l _task_name $_matched[2]
```

Comment:
😸 

I remove `if test -n $_matched` 
and I confirmed that it works without problems!!